File: InsertionSort.java
public class InsertionSort {
    public static void main(String[] args) {
        
        int arr[] = new int[]{9,15,14,12,6,8,13};
        
       for(int i=0;i<arr.length;i++){
        int j = i;
        while((j>0) && (arr[j-1] > arr[j])){
            int temp = arr[j];
            arr[j] = arr[j-1];
            arr[j-1] = temp; 
            j--;
        }
       }

       for(int k=0;k<arr.length;k++){
        System.out.print(arr[k]+" ");
       }
    }
}

File: InsertionSort.py
list = [29,10,14,37,13]

def insertionSort(list,n):
    i = 1
    while(i < n):
        key = list[i]
        j = i - 1
        while(j >= 0 and list[j] > key):
            list[j+1] = list[j]
            j -= 1 
        list[j+1] = key
        i += 1
    return

insertionSort(list,len(list))
print("Sorted list using insSort is : ",list)

    